### PR TITLE
Meenu/build: added prop for modal button color

### PR DESCRIPTION
**Pull request by meenakshi-deriv**

### CODE REVIEW

**Review by amina-deriv — APPROVED** [review]

File: lib/components/Input/InputGroup/Button/index.tsx [hunk 1]
```diff
@@ -21,6 +22,7 @@ export const InputGroupButton = forwardRef<
             button_position = "right",
             inputSize = "md",
             value,
+            buttonColor,
```
Comment (jia-deriv) [inline lib/components/Input/InputGroup/Button/index.tsx:25]: buttonColor = "black"

File: lib/components/Input/InputGroup/Button/index.tsx [hunk 2]
```diff
@@ -47,7 +49,7 @@ export const InputGroupButton = forwardRef<
         const InputButton = (
             <Button
                 disabled={isEmpty || disabled || buttonDisabled}
-                color="black"
+                color={buttonColor ? buttonColor : "black"}
```
Comment (jia-deriv) [inline lib/components/Input/InputGroup/Button/index.tsx:52]:
```suggestion
                color={buttonColor}
```

File: lib/components/Input/InputGroup/Button/index.tsx (hunk 1 above)
Comment (jia-deriv) [inline lib/components/Input/InputGroup/Button/index.tsx:25]:
```suggestion
            buttonColor = "black",
```

File: lib/components/Modal/base/index.tsx [hunk 3]
```diff
@@ -46,6 +47,7 @@ export const Modal = ({
     children,
     showHandleBar,
     showCrossIcon,
+    buttonColor,
```
Comment (jia-deriv) [inline lib/components/Modal/base/index.tsx]:
```suggestion
    buttonColor = "black-white",
```

File: lib/components/Modal/base/index.tsx [hunk 4]
```diff
@@ -171,7 +173,9 @@ export const Modal = ({
                     <div className="quill-modal__button-wrapper">
                         {showPrimaryButton && (
                             <Button
-                                color="black-white"
+                                color={
+                                    buttonColor ? buttonColor : "black-white"
```
Comment (jia-deriv) [inline lib/components/Modal/base/index.tsx]:
```suggestion
                                    buttonColor
```

**Review by jia-deriv — APPROVED** [review]

**Review by Nuzhy-Deriv — APPROVED** [review]

**Review by balakrishna-deriv — APPROVED** [review]

**Pull request merged**